template/template_example.Rmd

```diff
@@ -108,26 +105,31 @@ For more details on the contents of this folder see [these docs on refine.bio](h
 The `<experiment_accession_id>` folder has the data and metadata TSV files you will need for this example analysis.
 Experiment accession ids usually look something like `GSE1235` or `SRP12345`. 
 
-Copy and paste the `<experiment_accession_id>` folder into your newly created `data/` folder.
+Copy and paste the {{EXPERIMENT_ACCESSION_ID}} folder into your newly created `data/` folder.
 
 ## Check out our file structure!
 
 Your new analysis folder should contain: 
 
 - The example analysis Rmd you downloaded  
 - A folder called "data" which contains:
-  - The `<experiment_accession_id>` folder which contains:
+  - The `{{EXPERIMENT_ACCESSION_ID}}` folder which contains:
     - The gene expression  
     - The metadata TSV  
-
+- A folder for `plots` (currently empty)  
+- A folder for `results` (currently empty)  
+    
 Your example analysis folder should now look something like this (except with respective experiment accession id and analysis notebook name you are using): 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/b78046845763272d0aa65a077b6a3e46977d5342/template/screenshots/analysis-folder-structure.png" width=400>
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/dcce9a49246a15bfc7bb94a50a082c300c78074f/template/screenshots/analysis-folder-structure.png" width=400>
```

Comment:
```suggestion
<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/e140face75daa6d2c34e30a4755c362e6039a677/template/screenshots/analysis-folder-structure.png" width=400>
```

This would need to be updated to use the  updated screenshot you added to this PR.

Reply:
I don't know if changing the commit actually matters since the screenshots have not changed, but I can update them if you think its better. :+1:

Reply:
Ah the older version of the screenshot I have locally does not include the `plots` and `results` folders so I thought that was the part being updated in this PR.

Reply:
It's a commit from #197 I changed all the commit ids here since that is probably less messy.